packages/framework/src/Model/Customer/User/CustomerUserPasswordFacade.php

```diff
@@ -38,25 +39,33 @@ class CustomerUserPasswordFacade
      */
     protected $hashGenerator;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserRefreshTokenChainFacade
+     */
+    protected $customerUserRefreshTokenChainFacade;
+
     /**
      * @param \Doctrine\ORM\EntityManagerInterface $em
      * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserRepository $customerUserRepository
      * @param \Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface $encoderFactory
      * @param \Shopsys\FrameworkBundle\Model\Customer\Mail\ResetPasswordMailFacade $resetPasswordMailFacade
      * @param \Shopsys\FrameworkBundle\Component\String\HashGenerator $hashGenerator
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserRefreshTokenChainFacade $customerUserRefreshTokenChainFacade
      */
     public function __construct(
         EntityManagerInterface $em,
         CustomerUserRepository $customerUserRepository,
         EncoderFactoryInterface $encoderFactory,
         ResetPasswordMailFacade $resetPasswordMailFacade,
-        HashGenerator $hashGenerator
+        HashGenerator $hashGenerator,
+        CustomerUserRefreshTokenChainFacade $customerUserRefreshTokenChainFacade
```

Comment:
For me it is always better to use facade instead of repository, until there is cycling dependency between those. One of the reasons is that there might be some changes in the future, that will add some logic to the facade method.

Reply:
The point about changes in future is good to me.

Adding more dependencies always raise my attention and thinking about responsibilities.